comp/otelcol/ddflareextension/README.md

```diff
@@ -43,14 +43,22 @@ The flare will collect all environment variables, and these can be found in `ote
 The flare also adds data collected from extensions. These extensions are added automatically by the [converter component](../converter/README.md). The data collected is from extensions:
 - health_check: Found in `otel/otel-flare/health_check`. 
 
-Will contain a JSON of the latest health check, for example:
+Contains a JSON of the latest health check, for example:
 
 ```
 {"status":"Server available","upSince":"2024-08-14T14:54:00.575804+02:00","uptime":"28.470434291s"}
 ```
 - pprof: Found in `otel/otel-flare/pprof`
+
+Contains a allocs (`dd-autoconfigured_debug_pprof_allocs`), heap (`dd-autoconfigured_debug_pprof_heap`) and cpu (`dd-autoconfigured_debug_pprof_profile`) profile. Profiles can be opened with the pprof tool, e.g.
+```
+go tool pprof -http=: otel/otel-flare/pprof/dd-autoconfigured_debug_pprof_heap.dat
+```
```

Comment:
can u add local host link or link to pprof documentation

Reply:
Added link 👍